hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/replicas/ReplicasVerify.java

```diff
@@ -88,41 +110,130 @@ void findCandidateKeys(OzoneClient ozoneClient, OzoneAddress address) throws IOE
     String volumeName = address.getVolumeName();
     String bucketName = address.getBucketName();
     String keyName = address.getKeyName();
+
+    ObjectMapper mapper = new ObjectMapper();
+    ObjectNode root = mapper.createObjectNode();
+    ArrayNode keysArray = mapper.createArrayNode();
+
     if (!keyName.isEmpty()) {
       OzoneKeyDetails keyDetails = ozoneClient.getProxy().getKeyDetails(volumeName, bucketName, keyName);
-      processKey(keyDetails);
+      processKey(ozoneClient, keyDetails, keysArray);
     } else if (!bucketName.isEmpty()) {
       OzoneVolume volume = objectStore.getVolume(volumeName);
       OzoneBucket bucket = volume.getBucket(bucketName);
-      checkBucket(bucket);
+      checkBucket(ozoneClient, bucket, keysArray);
     } else if (!volumeName.isEmpty()) {
       OzoneVolume volume = objectStore.getVolume(volumeName);
-      checkVolume(volume);
+      checkVolume(ozoneClient, volume, keysArray);
     } else {
       for (Iterator<? extends OzoneVolume> it = objectStore.listVolumes(null); it.hasNext();) {
-        checkVolume(it.next());
+        checkVolume(ozoneClient, it.next(), keysArray);
       }
     }
+
+    root.set("keys", keysArray);
+    System.out.println(mapper.writerWithDefaultPrettyPrinter().writeValueAsString(root));
   }
 
-  void checkVolume(OzoneVolume volume) throws IOException {
+  void checkVolume(OzoneClient ozoneClient, OzoneVolume volume, ArrayNode keysArray) throws IOException {
     for (Iterator<? extends OzoneBucket> it = volume.listBuckets(null); it.hasNext();) {
       OzoneBucket bucket = it.next();
-      checkBucket(bucket);
+      checkBucket(ozoneClient, bucket, keysArray);
     }
   }
 
-  void checkBucket(OzoneBucket bucket) throws IOException {
+  void checkBucket(OzoneClient ozoneClient, OzoneBucket bucket, ArrayNode keysArray) throws IOException {
     for (Iterator<? extends OzoneKey> it = bucket.listKeys(null); it.hasNext();) {
       OzoneKey key = it.next();
       // TODO: Remove this check once HDDS-12094 is fixed
       if (!key.getName().endsWith("/")) {
-        processKey(bucket.getKey(key.getName()));
+        processKey(ozoneClient, bucket.getKey(key.getName()), keysArray);
       }
     }
   }
 
-  void processKey(OzoneKeyDetails keyDetails) {
-    replicaVerifiers.forEach(verifier -> verifier.verifyKey(keyDetails));
+  void processKey(OzoneClient ozoneClient, OzoneKeyDetails keyDetails, ArrayNode keysArray) {
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
To create the initial json object, use `JsonUtils#createObjectNode` which uses the default mapper.
- To add individual fields to this object, use `put`
- To add an object under this or another sub-object, use the return from `putObject`
- To add a list under this or another sub-object, use the return from `putArray`